src/Features/CSharp/Portable/Debugging/CSharpProximityExpressionsService_ExpressionTermCollector.cs

```diff
@@ -251,9 +251,11 @@ private static void AddObjectCreationExpressionTerms(ObjectCreationExpressionSyn
 
         private static void AddInvocationExpressionTerms(InvocationExpressionSyntax invocationExpression, IList<string> terms, ref ExpressionType expressionType)
         {
+#pragma warning disable IDE0059 // Unnecessary assignment of a value
             // Invocations definitely have side effects.  So we assume this
             // is invalid initially;
             expressionType = ExpressionType.Invalid;
+#pragma warning restore IDE0059 // Unnecessary assignment of a value
```

Comment:
why?

Reply:
I am actually not sure if this would cause any side effects, especially given the comment. Just playing safe.